system-int-test/test/cmr/system_int_test/search/collection_search_format_test.clj

```diff
@@ -10,6 +10,7 @@
    [cmr.common-app.config :as common-config]
    [cmr.common-app.test.side-api :as side]
    [cmr.common.mime-types :as mt]
+   [cmr.common.test.url-util :as uu]
```

Comment:
We've all decided to use long names going forward for aliases (`mt` above should be `mime-types`, this should `url-util`).